config/env.js

```diff
@@ -1,7 +1,7 @@
-const environment = {
-  port: process.env.PORT || 2022,
-  nodeEnv: process.env.NODE_ENV || "development",
-  mongodbUri: process.env.DB_URI || "mongodb://127.0.0.1:27017/boilerplate",
-};
-
-export default environment;
+const environment = {
+  port: process.env.PORT || 2022,
+  nodeEnv: process.env.NODE_ENV || "development",
+  mongodbUri: process.env.DB_URI || "mongodb://127.0.0.1:27017/HRMS",

```

Comment:
use Live MongoDb

Reply:
Aoa,sir according to the documentation we've to use MongoDB compass but as you've mentioned using MongoDB live you're referring to the Atlas version ??